pkg/client/unversioned/clientcmd/api/types.go

```diff
@@ -68,6 +69,8 @@ type Cluster struct {
 	InsecureSkipTLSVerify bool `json:"insecure-skip-tls-verify,omitempty"`
 	// CertificateAuthority is the path to a cert file for the certificate authority.
 	CertificateAuthority string `json:"certificate-authority,omitempty"`
+	// VersionLabels
```

Comment:
Comment should explain what this is for.  VersionLabels isn't very descriptive, and it is possible there are labels we want to drop from the service label selector that aren't about versioning.  Maybe `ReservedLabels`?